src/lib.rs

```diff
@@ -183,10 +204,11 @@ pub fn render_password(entropy: &[u8], charset: CharacterSet, len: u8) -> String
 }
 
 /// Return the SHA-256 fingerprint that corresponds to the given master password.
-pub fn get_fingerprint(password: &str) -> hmac::Signature {
-    let key = hmac::SigningKey::new(&digest::SHA256, password.as_bytes());
-
-    hmac::sign(&key, b"")
+pub fn get_fingerprint(password: &str) -> Vec<u8> {
+    let mut mac = Hmac::<Sha256>::new_varkey(password.as_bytes())
+        .expect("Hmac's new_varkey implementation is infallible");
```

Comment:
If it never fails, can you instead match on it and use `unreachable!()` in the `Err` branch?

Reply:
What you are suggesting is literally the implementation of `Result::unwrap`, and `Result::expect` is the same as `Result::unwrap` with a more meaningful error message.

Reply:
I find `unreachable` more meaningful than `expect` / `unwrap`, since these can mean "there might be an error, and if there is I will panic", whereas `unreachable` specifically means "the API specifies that a `Result` is used here, but for this particular implementation this will never happen."

Reply:
You never unwrap things that can fail, the only legitimate reason to use unwrap in code other than tests is because you know it is infallible.  `expect` is the nicer version, because you can inform anyone reading the code why it is safe to unwrap the result in question.

What I have done is the idiomatic thing to do.

I fear that if we cannot find a common ground on trivial issues like this, then I don't see us collaborating on larger changes.  And that is okay, I just need to know.

Reply:
I still think that `unreachable` is more meaningful in those cases, but you're right to say that `expect` is more idiomatic. Please add a short doc comment above and the PR will be ready to go!